src/components/routeComponents/memberRegistrationCorrectionForm/AssignLevelAndClassroomLink.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const LinkStyled = styled.a`
+	color: #f9570a;
+	font-weight: 500;
+	font-size: 14px;
+	line-height: 21px;
+	border-bottom: 1px solid #f9570a;
+	&:hover {
+		color: darkorange;
+		cursor: pointer;
+	}
+`;
+
+const AssignLevelAndClassroomLink = ({ constants, onClickShowAssignLevelAndClassroomModal }) => {
```

Comment:
Pass student data as well, So that we can display assigned level and classroom here
```suggestion
const AssignLevelAndClassroomLink = ({ constants, onClickShowAssignLevelAndClassroomModal, student }) => {
```